### PR TITLE
[RHACS] ROX-20537: Updating docs for audit log to reflect new source IP field

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.3+

Issue:
https://issues.redhat.com/browse/ROX-20537

Link to docs preview:
https://67405--docspreview.netlify.app/openshift-acs/latest/configuration/configure-audit-logging#sample-audit-log-message_configure-audit-logging

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs` and cherry-pick to:
    - `rhacs-docs-4.3`

### CODE REVIEW

**Comment by agantony** [comment]
Tech review: @dhaus67

**Review by dhaus67 — APPROVED** [review]

File: modules/sample-audit-log-message.adoc
Comment (xenolinux) [inline modules/sample-audit-log-message.adoc:67]:
```suggestion
The source IP address of the request displayed in the source fields makes it easier to investigate audit log requests and identify their origin.
```

Comment (xenolinux) [reply modules/sample-audit-log-message.adoc:67]: Up to you...

**Review by xenolinux — COMMENTED** [review]
I left a few suggestions but otherwise looks good!
Can you please provide the preview link and squash commits?

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

File: modules/sample-audit-log-message.adoc
Comment (agantony) [reply modules/sample-audit-log-message.adoc:79]: "it" refers to Central, @dhaus67 Please correct me if I am wrong?

Comment (abhatt-rh) [inline modules/sample-audit-log-message.adoc:79]:
```suggestion
* If you expose Central behind a load balancer, for example, if you are running it on Google Kubernetes Engine (GKE) or Amazon Elastic Kubernetes Service (Amazon EKS) by using the Kubernetes External Load Balancer service type, see link:https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip[Preserving the client source IP].
```

File: modules/sample-audit-log-message.adoc
Comment (abhatt-rh) [reply modules/sample-audit-log-message.adoc:69]: In the context of APIs, I'd be inclined to go with the term _parameters_ instead of fields. Whichever, you use, be consistent with the use across the section.

File: modules/sample-audit-log-message.adoc
Comment (abhatt-rh) [reply modules/sample-audit-log-message.adoc:67]: In case you use the term _parameters_: 
```suggestion
The source IP address of the request is displayed in the source parameters, which makes it easier for you to investigate audit log requests and identify their origin.
```

File: modules/sample-audit-log-message.adoc
Comment (abhatt-rh) [reply modules/sample-audit-log-message.adoc:81]: In case you use the term _parameters_:
```suggestion
* If you expose Central with a TLS passthrough route, you can not determine the source IP address of the client. A cluster-internal IP address is displayed in the source parameters as the source IP address of the client.
```

**Review by abhatt-rh — COMMENTED** [review]
@agantony and @xenolinux +1 to peer review. Added a few additional nits. 
As Serversha mentioned, kindly add a link to the docs preview and squash your commits before you request a merge

File: modules/sample-audit-log-message.adoc
Comment (agantony) [reply modules/sample-audit-log-message.adoc:81]: @dhaus67 Can we go ahead with this suggestion.

Comment (dhaus67) [reply modules/sample-audit-log-message.adoc:81]: Yeah, parameters is fine as well.

File: modules/sample-audit-log-message.adoc
Comment (dhaus67) [reply modules/sample-audit-log-message.adoc:79]: Yeah, `it` does refer to the Central instance.

**Pull request merged**
